tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>"Before you use the clustering API, you must define and train the model" </p><p> </p><p>define and train the model is during usage of the clustering API, not before right? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<blockquote>Before you use the clustering API, you must define and train the model</blockquote><p>What I meant was clustering API works on a pre-trained baseline model. So I changed this sentence into:</p><blockquote>Before you use the clustering API, you should train the baseline model and see some acceptable accuracy</blockquote>